index.html

```diff
@@ -115,6 +115,39 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+
+<!-- ________ Mocktic Joshi's Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Mocktic Joshi</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/mocktic" target="_blank">mocktic</a>
+          </p>
+          <p class="about">I love to code and build something fot the community</p>
```

Comment:
Found a typo in this sentence. I think you meant to use the word "for" not "fot" :)